massa-node/base_config/config.toml

```diff
@@ -1,96 +1,175 @@
     future_block_processing_max_periods = 100
+    # max number of blocks in the future kept in RAM
     max_future_processing_blocks = 400
+    # max number of blocks waiting for dependencies
     max_dependency_blocks = 2048
+    # number of final periods that must be kept at all time (increse to more resilience to short network disconnections)
+    force_keep_final_periods = 20
+
+    # max number of time the node tries to fill up a block before producing it
     max_operations_fill_attempts = 102400
+    # number of operation to include in a block
+    operation_batch_size = 1024
+
+    # path to the cold sored ledger
     ledger_path = "storage/ledger"
+    # size of the ledger kept in cache
     ledger_cache_capacity = 1048576
+    # [optionnal] the ledger is saved on disk every ledger_flush_interval
     ledger_flush_interval = 10000
+    # wether to overwrite the ledger at start
     ledger_reset_at_startup = true
+    # path to the initial ledger
     initial_ledger_path = "base_config/initial_ledger.json"
-    operation_batch_size = 1024
+
+    # max millis to wait while sending an event before dropping it
     max_send_wait = 500
-    force_keep_final_periods = 20
+    # useless blocks are pruned every block_db_prune_interval ms
+    block_db_prune_interval = 5000
+
+    # path to the initial roll distribution
     initial_rolls_path = "base_config/initial_rolls.json"
+    # number of cached cycle for proof of stake seed initialization
     pos_draw_cached_cycles = 10
+
+    # path to your staking private keys (not encrypted)
     staking_keys_path = "config/staking_keys.json"
+    # considered timespan for stats info
     stats_timespan = 60000
-    block_db_prune_interval = 5000
+    # max number of item returned per query
     max_item_return_count = 100
 
 [protocol]
+    # timeout after which without answer a hanshake is ended
     message_timeout = 5000
+    # timeout after whick we consider a node does not have the block we asked for
     ask_block_timeout = 10000
+    # max number for the local vision of what blocks a node knows
     max_node_known_blocks_size = 1024
+    # max number for the local vision of what blocks a node wants
     max_node_wanted_blocks_size = 1024
+    # max number of blocks we can asked per node
     max_simultaneous_ask_blocks_per_node = 2048
+    # max millis to wait while sending an event before dropping it
     max_send_wait = 500
+    # max number for the local vision of what operations a node knows
     max_known_ops_size = 10024
+    # max number for the local vision of what endorsements a node knows about
     max_known_endorsements_size = 1024
 
 [network]
+    # where to listen for communication
     bind = "[::]:31244"
+    # port used by protocol
     protocol_port = 31244
+    # millis spent waiting for a respone
     connect_timeout = 3000
+    # connection attempt to avaible peers every wakeup_interval millis
```

Comment:
attempt a connection to available peers when needed every wakeup_interval milliseconds